content/posts/0.20-release.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Pyodide 0.12.0 release"
+date: 2022-01-10
+draft: true
+tags: ["release", "announcement"]
+author: "Hood Chatham on behalf of the Pyodide team"
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+# description: ""
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+---
+
+Pyodide 0.20 is a major release focused on improving our system for building and
+packaging Python modules. 
+
+We upgraded to Python 3.10. This comes with a speed boost of around 15% in
+Firefox and 22% in Chrome.
+
+We updated almost every package to the most recent release, and moved closer to
+using upstream Emscripten. Pyodide 0.20 also includes many small bug fixes and
+usability improvements.
+
+## Build system
+
+We dramatically reworked our build system backend. The result is simpler, more
+powerful, and less error prone. 
+
+We now produce Python wheels for packages instead of Emscripten file packager
+archives. These are much easier to work with: they can be processed with any
+software that handles zip files and also by all of the Python tools for handling
+wheels. For most pure Python packages, we now use their wheels directly from
+PyPI.
+
+We updated our system to use `pypa/build`, including (mostly) isolated builds.
+This way packages can use custom build systems, including different versions of
+setuptools.
+
+We added support for C++ exception handling in packages and made other many
+minor tweaks to make it easier to port packages.
+
+## Upstreaming patches into Python 3.11
+
+We worked with the CPython upstream to include our patches into Python 3.11.
+Experimental builds of Pyodide based on tip-of-tree Python confirm that
+everything works without patches.
+
+## Emscripten Update
+
+We upgraded from Emscripten 2.0.16 to Emscripten 2.0.27. We are still a fair
+distance behind the newest Emscripten release due to a bug in Firefox versions
+95 -- 98 but fixed in Firefox 99beta. 
+
+Emscripten 2.0.27 is a major improvement over Emscripten 2.0.16, particularly
+with regards to linking: many linker errors have been changed from call-time
+errors to load-time errors or compile-time errors. In particular, this allowed
+us to fix tons of Scipy bugs.
+
```

Comment:
It would be good to add a link to the detailed changelog somewhere

Reply:
And from the changelog to here.